phpunit.xml.dist

```diff
@@ -1,35 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         bootstrap="vendor/autoload.php"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
-         verbose="true"
->
-    <testsuites>
-        <testsuite name="Tipoff Test Suite">
-            <directory>tests</directory>
-        </testsuite>
-    </testsuites>
-    <coverage>
-        <include>
-            <directory suffix=".php">./src</directory>
-        </include>
-        <report>
-            <html outputDirectory="build/coverage"/>
-            <text outputFile="build/coverage.txt"/>
-            <clover outputFile="build/logs/clover.xml"/>
-        </report>
-    </coverage>
-    <logging>
-        <junit outputFile="build/report.junit.xml"/>
-    </logging>
-    <php>
-        <env name="DB_CONNECTION" value="testing"/>
-        <env name="APP_KEY" value="AckfSECXIvnK5r28GVIWUAxmbBSjTsmF"/>
-    </php>
-</phpunit>
+    <phpunit backupGlobals="false"
+             backupStaticAttributes="false"
+             bootstrap="vendor/autoload.php"
+             colors="true"
+             convertErrorsToExceptions="true"
+             convertNoticesToExceptions="true"
+             convertWarningsToExceptions="true"
+             processIsolation="false"
+             stopOnFailure="false"
+             verbose="true"
+    >
+        <testsuites>
+            <testsuite name="Tipoff Test Suite">
+                <directory>tests</directory>
+            </testsuite>
+        </testsuites>
+        <coverage>
+            <include>
+                <directory suffix=".php">./src</directory>
+            </include>
+            <report>
+                <html outputDirectory="build/coverage"/>
+                <text outputFile="build/coverage.txt"/>
+                <clover outputFile="build/logs/clover.xml"/>
+            </report>
+        </coverage>
+        <logging>
+            <junit outputFile="build/report.junit.xml"/>
+        </logging>
+        <php>
+            <env name="DB_CONNECTION" value="testing"/>
+            <env name="APP_KEY" value="AckfSECXIvnK5r28GVIWUAxmbBSjTsmF"/>
+        </php>
+    </phpunit>
```

Comment:
@huntermontell Let's make sure every package has a phpunit file that looks like this.